go/oasis-node/cmd/common/grpc/grpc.go

```diff
@@ -74,7 +74,11 @@ func NewServerLocal(installWrapper bool) (*cmnGrpc.Server, error) {
 func NewClient(cmd *cobra.Command) (*grpc.ClientConn, error) {
 	addr, _ := cmd.Flags().GetString(CfgAddress)
 
-	conn, err := grpc.Dial(addr, grpc.WithInsecure()) // TODO: TLS?
+	conn, err := grpc.Dial(
```

Comment:
Should the `defaultAddress` be switched to the internal socket at some point?

Reply:
This is impossible as you don't know the path. Could just be `internal.sock` if that's what you mean?

Reply:
I'll just set it to `"unix:"+localSocketFilename`.

Reply:
Done.

Reply:
oh right, you would be missing the `--datadir`. I'm fine with either then, maybe no default would actually be the best in that case?